addons/medical_treatment/ACE_Medical_Treatment.hpp

```diff
@@ -565,8 +565,11 @@ class ADDON {
         timeInSystem = 120;
         // How long until the maximum effect is reached
         timeTillMaxEffect = 30;
-        // How many of this type of medication can be in the system before the patient overdoses?
+        // How many of this type of medication can be in the system before the patient can possibly overdose?
         maxDose = 4;
+        // The number of doses over maxSafeDose where there is a chance to avoid overdose. Chance decreases as dose gets higher.  
```

Comment:
Currently the last sentence here is not true (see my other comment).

Reply:
The last sentence might be misleading: On one hand, the distribution is uniform, so the chance doesn't change, but on the other, if you consider the increase in dosage, the chance of you overdosing increases.

I'm not sure how to improve it though, imo it's clear enough for the average user.